python3-diff.md

```diff
@@ -0,0 +1,42 @@
+# Differences between Python 3 and Starlark
+
+Starlark language is very similar to Python. Starlark tries to follow Python
+syntax and semantics where possible, but full compatibility with Python is neither possible nor a goal.
+
+This document describes some differences between Python 3 and Starlark.
+The list is incomplete.
+
+This document is not normative.
+
+## The differences
+
+* Starlark evaluation is deterministic and should produce the identical output in any implementation
```

Comment:
> deterministic

This is true of the the core language, data structures, and library, but applications may add nondeterministic features.


> identical output

This is not possible because of differences in string representation (UTF-8 vs 16), and possibly for other reasons. It is a guiding principle but not an actual invariant.